source/server/nierr_Status.cpp

```diff
@@ -0,0 +1,106 @@
+/*!
+   \file nierr_Status.c
+   \brief Status C functions, the low level manipulator for Status
+*/
+/*
+   Copyright (c) 2012-2012 National Instruments Corporation.
+   All rights reserved.
+*/
+
+
+/* includes... */
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "nierr_Status.h"
```

Comment:
The `.cpp` should always include the corresponding `.h` file first, in order to ensure that the `.h` file isn't missing any declarations or includes.